lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -120,7 +137,16 @@ def check_file_request(file, klass, validate_all, lines_to_validate, must_suppor
               end
             end
           end
-          assert errors.empty?, "Failed Profile validation for resource #{line_count}: #{errors}"
+
+          error_collection[line_count] = errors unless errors.empty?
+        end
+
+        unless error_collection.empty?
+          response_for_log[:body] = line_collection.join
+          LoggedRestClient.record_response(request_for_log, response_for_log)
+
+          index, errors = error_collection.first
+          assert false, "#{error_collection.size} / #{line_count} #{klass} resources failed profile validation. The first failed is #{klass} ##{index}: #{errors}"
```

Comment:
Just a small phrasing update
```suggestion
          assert false, "#{error_collection.size} / #{line_count} #{klass} resources failed profile validation. The first failed resource is #{klass} ##{index}: #{errors}"
```